packages/bodiless-demo/src/components/Video/AutoPlayback.tsx

```diff
@@ -0,0 +1,44 @@
+// from https://esausilva.com/2021/06/14/react-hook-to-play-video-using-intersection-observer/
+
+import { useRef, useEffect } from 'react';
+
+// @ts-ignore:  TS7006: Parameter 'entries' implicitly has an 'any' type.
+const useVideoAutoPlayback = options => {
+  const containerRef = useRef<HTMLDivElement>(null);
+  const videoRef = useRef<HTMLVideoElement>(null);
+
+  // catching the unmounted error
+  if (!containerRef || !videoRef) return [null, null];
+
+  // @ts-ignore:  TS7006: Parameter 'entries' implicitly has an 'any' type.
+  const cb = entries => {
+    const [entry] = entries;
+    if (videoRef.current) {
+      if (entry.isIntersecting) {
+        // https://developer.chrome.com/blog/play-request-was-interrupted/
+        const playPromise = videoRef.current!.play();
```

Comment:
ERROR 3:

Issue: in edit interface -- if I clicked add component then it threw a dom exception The play() request was interrupted by a call to pause().

add the try/check on play at lines 18 - 30 per doc